dotnet/src/Agents/Core/Chat/JsonResultTranslator.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+using System.Text.Json;
+
+namespace Microsoft.SemanticKernel.Agents.Chat;
+/// <summary>
+/// Supports parsing json from a text block that may contain literals delimiters:
+/// <example>
+/// [json]
+/// </example>
+/// <example>
+/// ```
+/// [json]
+/// ```
+/// </example>
+/// <example>
+/// ```json
+/// [json]
+/// ```
+/// </example>
+/// </summary>
+/// <remarks>
+/// Encountering json with this form of delimiters is not uncommon for agent scenarios.
+/// </remarks>
+public static class JsonResultTranslator
```

Comment:
Do we need this as part of the public API surface or can it be a sample?

Reply:
Its a common pattern to strip literal delimiters from the JSON result, but I can move it to the sample.  We can always decided to share the utility with users at some later point should we realize the value.

Reply:
Moved to samples